src/app/server/OnboardingCodesUtil.cpp

```diff
@@ -67,6 +67,39 @@ void PrintOnboardingCodes(chip::RendezvousInformationFlags aRendezvousFlags)
     }
 }
 
+void PrintOnboardingCodes(const chip::SetupPayload & payload)
+{
+    std::string QRCode;
```

Comment:
nit: `qrCode`

Reply:
I was thinking of it, but still keep this name to align with other places in the current code base